posthog/api/session_recording.py

```diff
@@ -140,13 +140,18 @@ def delete(self, request: request.Request, *args: Any, **kwargs: Any) -> Respons
 
     @action(methods=["GET"], detail=True)
     def snapshot_file(self, request: request.Request, **kwargs) -> HttpResponse:
+        recording = SessionRecording.get_or_build(session_id=kwargs["pk"], team=self.team)
+
+        if recording.deleted:
+            raise exceptions.NotFound("Recording not found")
+
```

Comment:
I wonder if we should make this block a standard thing for all the detail routes 🤔

Reply:
It will eat at my soul that we use `recording.deleted` to mean not found but I'll cover in a follow-up 🤣